src/modules/lorem/index.ts

```diff
@@ -332,7 +358,26 @@ export class LoremModule {
 
     const method = this.faker.helpers.arrayElement(methods);
 
-    return `${this[method]()}`;
+    if (typeof options === 'number') {
+      options = { length: options };
+    }
+
+    if (options.length == null) {
+      return `${this[method]()}`;
+    }
+
+    const length = this.faker.helpers.rangeToNumber(options.length);
+
+    if (length <= 0) {
+      return '';
+    }
+
+    let text = '';
+    do {
+      text = `${text}${this[method]()} `;
```

Comment:
If we go for `sentence()` (not sure yet):

```suggestion
      text = `${text}${this.sentence()} `;
```